shunit2_asserts_test.sh

```diff
@@ -246,6 +246,110 @@ testAssertFalse() {
   th_assertFalseWithError 'too many arguments' $? "${stdoutF}" "${stderrF}"
 }
 
+testAssertFileExists() {
+  ( assertFileExists "$TH_EXISTING_FILE" >"${stdoutF}" 2>"${stderrF}" )
```

Comment:
Rather than using a variable (i.e. ${TH_EXISTING_FILE}), be explicit (see comment later) about which file you are testing for. This makes the code easier to read, and easier to debug.

E.g., if you chose the name 'this_file_exists', then use exactly that filename in the test rather than a variable.

With unit tests, it is best not to "hide" info inside of variables as it makes it more difficult to read the test, and definitely more difficult to debug as one cannot search for the failed strings as easily.